src/widget/align.rs

```diff
@@ -21,10 +21,16 @@ use crate::{
 
 use crate::piet::UnitPoint;
 
+enum Axis {
```

Comment:
I have a question about the axis option for Align, as Flutter doesn't have precisely that. Is Axis::Horizontal roughly equivalent to setting `heightFactor = 1.0` in Flutter?

(I asked this in Zulip and the answer seems to be yes, but wanted to leave a record in the PR)